docs/teams.md

```diff
@@ -0,0 +1,30 @@
+# Teams BETA
```

Comment:
"Teams (Beta)"

And somewhere.
Please [contact us](mailto:sales@quiltdata.io) to join the Beta.